mne_bids/write.py

```diff
@@ -289,6 +289,39 @@ def _events_tsv(events, durations, raw, fname, trial_type, overwrite=False):
     _write_tsv(fname, data, overwrite)
 
 
+def _events_json(fname, overwrite=False):
+    """Create participants.json for non-default columns in accompanying TSV.
+
+    Parameters
+    ----------
+    fname : str | mne_bids.BIDSPath
+        Output filename.
+    overwrite : bool
+        Whether to overwrite the output file if it exists.
+    """
+    new_data = {
+        'sample': {
+            'Description': 'The event onset time in number of sampling points.'
+        },
+        'value': {
+            'Description': (
+                'The event code / trigger code associated with the event.'
+            )
+        }
+    }
+
+    # make sure to append any JSON fields added by the user
+    fname = Path(fname)
+    if fname.exists():
+        orig_data = json.loads(
+            fname.read_text(encoding='utf-8'),
+            object_pairs_hook=OrderedDict
+        )
+        new_data = {**orig_data, **new_data}
+
+    _write_json(fname, new_data, overwrite)
```

Comment:
Code duplication from below. But I felt it wasn't worth creating a new helper function for this. thoughts?